python-package/MANIFEST.in

```diff
@@ -1,6 +1,6 @@
 prune build
 include *.rst *.txt
-recursive-include lightgbm *.py *.txt *.so
+recursive-include lightgbm *.py *.txt *.so LICENSE
```

Comment:
since you copy the license to the ```python-package``` folder (not the ```python-package/lightgbm``` folder). I think you should include it in the line 2 of this file.